tests/framework/util.go

```diff
@@ -68,6 +68,17 @@ func RunOcDescribeCommand(resourceType, resourceName string) string {
 	return execute(Result{cmd: "oc", verb: "describe", resourceType: resourceType, resourceName: resourceName})
 }
 
+func GetVirtualMachineSpecificParameters(resourceType, resourceName, query string) string {
```

Comment:
Is this only for VM?

Reply:
renamed to GetResourceSpecificParameters as It is necessary not only for VM